maven/bnd-resolver-maven-plugin/pom.xml

```diff
@@ -84,7 +88,7 @@
                       <jsse.enableSNIExtension>false</jsse.enableSNIExtension>
                     </properties>
                     <goals>
-                        <goal>bnd-resolver:resolve</goal>
+                        <goal>package</goal>
```

Comment:
This is only so that the test can trigger the target which is not bound by default.